test/e2e/kubevirt.go

```diff
@@ -1317,11 +1317,7 @@ passwd:
 
 			step := by(vmi.Name, "Login to virtual machine for the first time")
 			Eventually(func() error {
-				if td.role != "primary" {
-					return kubevirt.LoginToFedora(vmi, "core", "fedora")
-				} else {
-					return kubevirt.LoginToFedoraWithHostname(vmi, "core", "fedora", "localhost")
-				}
```

Comment:
why did this binding type changed the prompt ?

Reply:
since with the new one the DHCP populate now the right hostname, no need this special treatment that was to workaround this when passt is used
Quique is familiar with it best, and said it is good btw

Reply:
to me, it seems like we first need to decide if we want to keep both bindings around or not. 

like, if we keep passt around, we can't have this change.

Reply:
those are the tests, in tests as long as we dont test both passt and managedTap we can have just one
it is orthogonal to have both on the CR and then the user can still choose

Reply:
just to add some context, passt do not support the DHCPv4 option hostname (12), so VM ends up with localhost, also is good that we test this kind of things since at managedTap we pass that with the OVN DHCP.

Reply:
sgtm